itest/lnd_sweep_test.go

```diff
@@ -700,9 +700,9 @@ func testSweepHTLCs(ht *lntest.HarnessTest) {
 	cltvDelta := routing.MinCLTVDelta
 
 	// Start tracking the deadline delta of Bob's HTLCs. We need one block
-	// for the CSV lock, and another block to trigger the sweeper to sweep.
-	outgoingHTLCDeadline := int32(cltvDelta - 2)
-	incomingHTLCDeadline := int32(lncfg.DefaultIncomingBroadcastDelta - 2)
+	// to trigger the sweeper to sweep.
```

Comment:
yeah correct, it's already accounted as we had -2 before so we now use -1. In addition, we need this because the fee func start at 0, then ends at `cltvDelta - 1`, which means the deadline delta is still `cltvDelta`.